### PR TITLE
DX: Clean ups

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: tests/Indicator/PhpUnitTestCaseIndicatorTest.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+use PhpCsFixer\Indicator\PhpUnitTestCaseIndicator;
+use PhpCsFixer\Tests\TestCase;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ * @coversNothing
```
Comment (SpacePossum) [inline tests/Indicator/PhpUnitTestCaseIndicatorTest.php]: not sure at this one, cc @keradus

Comment (keradus) [reply tests/Indicator/PhpUnitTestCaseIndicatorTest.php]: why not `$covers \PhpCsFixer\Indicator\PhpUnitTestCaseIndicator` ?

Comment (SpacePossum) [reply tests/Indicator/PhpUnitTestCaseIndicatorTest.php]: sounds good! I'm just not to familiar with these annotations I think

**Comment by SpacePossum** [comment]
haha sorry at that organisation (great name btw :P), is there a way to escape such an annotation in a commit message?

**Comment by keradus** [comment]
for my best knowledge, nope

**Comment by SpacePossum** [comment]
rebased and forced pushed after resolving conflict

File: tests/AutoReview/FixerFactoryTest.php [hunk 2]
```diff
@@ -62,7 +62,6 @@ public function provideFixersPriorityCases()
             [$fixers['array_indentation'], $fixers['binary_operator_spaces']],
             [$fixers['array_syntax'], $fixers['binary_operator_spaces']],
             [$fixers['array_syntax'], $fixers['ternary_operator_spaces']],
-            [$fixers['backtick_to_shell_exec'], $fixers['escape_implicit_backslashes']],
```
Comment (keradus) [inline tests/AutoReview/FixerFactoryTest.php]: could you elaborate on this one? are we sure there is no prioritiy conflict between them ?

I assume you didn't saw warning from https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.11/tests/IntegrationTest.php#L71 
(as if you would and willingly drop the test here, you would also drop the expection there)

Comment (keradus) [reply tests/AutoReview/FixerFactoryTest.php]: Also, `backtick_to_shell_exec` has
> // Should run before escape_implicit_backslashes

comment in `getPriority`

Comment (SpacePossum) [reply tests/AutoReview/FixerFactoryTest.php]: on second thought I revered the change, making it more easy to pass this PR

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**